### PR TITLE
Snapdragon: updating the ADSP and troubleshooting

### PR DESCRIPTION
**Pull request by julianoes**
This brings improvements on how to update the Linux and DSP side of the Snapdragon.

Also, it adds some troubleshooting info.

Can be merged if the syntax is correct and all the links work.

### CODE REVIEW

**Comment by LorenzMeier** [comment]
@julianoes Links don't work

File: book/advanced-snapdragon.md [hunk 1]
```diff
@@ -196,6 +313,33 @@ This can happen if ramdumps fill up the disk. To clean up, do:
...
 
+Also, the logs might have filled the space. To delete them, do:
+
+```
+rm -rf /root/logs/*
+```
+
+### Undefined PLT symbol
+
+#### _FDtest
```
Comment (LorenzMeier) [inline book/advanced-snapdragon.md]: @julianoes I would guess this is the underscore breaking it.

Comment (julianoes) [inline book/advanced-snapdragon.md]: No, this one seems ok. It was the references containing a `/`.

**Comment by julianoes** [comment]
Ok, two links are now just not urls anymore. Will follow up with a fix.

**Pull request merged**